src/main/java/network/brightspots/rcv/StreamingCVRReader.java

```diff
@@ -100,9 +100,11 @@
     // to keep our code simple, we convert 1-indexed user-supplied values to 0-indexed here
     this.firstVoteColumnIndex = source.getFirstVoteColumnIndex() - 1;
     this.firstVoteRowIndex = source.getFirstVoteRowIndex() - 1;
-    this.idColumnIndex = source.getIdColumnIndex() != null ? source.getIdColumnIndex() - 1 : null;
+    this.idColumnIndex = source.getIdColumnIndex() != null && !source.getIdColumnIndex().isBlank() ?
```

Comment:
Yes, for safety, and because no downside. Been waiting a long time for us to get moved to JDK 11 to do this. Great idea on the utility method, I'll do that.